game/src/js/objects/ui/Button.js

```diff
@@ -1,21 +1,20 @@
 export default class Button extends Phaser.GameObjects.Text {
-  constructor(scene, x, y, text, options = {}) {
-    super(scene, x, y, text, {
-      fontSize: 30,
-      fontFamily: "'Fjalla One', sans-serif"
-    });
-
-    this.props = {
+  constructor(scene, x, y, text, options = {
```

Comment:
This default parameter value will be overwritten when an object is passed. It will only be used when no value at all is passed in. There's no way to have partial defaults now.